DeanerySystem/DeanerySystem.WebUI/Startup.cs

```diff
@@ -1,18 +1,20 @@
-﻿using DeanerySystem.DataAccess.Concrete;
+﻿using DeanerySystem.DataAccess.Abstract;
+using DeanerySystem.DataAccess.Concrete;
 using DeanerySystem.DataAccess.Entities.Identity;
+using DeanerySystem.DataAccess.Utilities;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Identity;
 using Microsoft.EntityFrameworkCore;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
-using DeanerySystem.WebUI.Models;
 using DeanerySystem.WebUI.Services;
 
 namespace DeanerySystem.WebUI
 {
     public class Startup
     {
+        private const bool UseSqlDb = true;  
```

Comment:
`UseSqlDb` is used in several `if` statements will it ever be not `true`?

Reply:
Yes, I case you would like to work with InMemory database  (Mocks)

Reply:
But how you will be able to set a different value for this variable?